clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -3408,6 +3408,8 @@ def warn_objc_redundant_literal_use : Warning<
 def err_attr_tlsmodel_arg : Error<"tls_model must be \"global-dynamic\", "
   "\"local-dynamic\", \"initial-exec\" or \"local-exec\"">;
 
+def err_attr_codemodel_arg : Error<"code_model '%0' is not supported on this target">;
```

Comment:
`code_model` is LLVM-speak, not C; the user doesn't write `code_model` in the C code. "code model" may sound better.

Reply:
Done. Thanks